common/src/main/java/com/wynntils/utils/mc/ComponentUtils.java

```diff
@@ -114,7 +115,13 @@ public static List<Component> splitComponent(Component component, int maxWidth)
         return split;
     }
 
-    public static MutableComponent makeRainbowStyle(String name) {
+    public static MutableComponent makeRainbowStyle(String name, boolean useRainbowEffect) {
+        if (useRainbowEffect) {
```

Comment:
```suggestion
    public static MutableComponent makeRainbowStyle(String name, boolean useRainbowFont) {
        if (useRainbowFont) {
```

Do you think this is clearer?

Reply:
Hmm, it's not really a font, but I do agree "effect" isn't really the best either. I'll think about it

Reply:
`useRainbowStyle`?

Reply:
After commenting, I saw the method name. :-D 

`useFontStyle`?

Reply:
Maybe `useRainbowShader` as it's achieved via shaders in the resource pack?

Reply:
Or just `useShader`? The rainbow part is a bit redundant in this method...

Reply:
That's true, I'll go with that